nanodbc/nanodbc.cpp

```diff
@@ -90,6 +90,37 @@
 #define SQL_SS_UDT (-151) // from sqlncli.h
 #endif
 
+#ifndef SQL_SOPT_SS_BASE
+#define SQL_SOPT_SS_BASE 1225
+#endif
+#ifndef SQL_SOPT_SS_PARAM_FOCUS
+#define SQL_SOPT_SS_PARAM_FOCUS (SQL_SOPT_SS_BASE + 11)
+#endif
+#ifndef SQL_SOPT_SS_NAME_SCOPE
+#define SQL_SOPT_SS_NAME_SCOPE (SQL_SOPT_SS_BASE + 12)
+#endif
+#ifndef SQL_SS_NAME_SCOPE_TABLE
+#define SQL_SS_NAME_SCOPE_TABLE 0
+#endif
+#ifndef SQL_SS_NAME_SCOPE_TABLE_TYPE
+#define SQL_SS_NAME_SCOPE_TABLE_TYPE 1
+#endif
+#ifndef SQL_SS_NAME_SCOPE_EXTENDED
+#define SQL_SS_NAME_SCOPE_EXTENDED 2
+#endif
+#ifndef SQL_SS_NAME_SCOPE_SPARSE_COLUMN_SET
+#define SQL_SS_NAME_SCOPE_SPARSE_COLUMN_SET 3
+#endif
+#ifndef SQL_SS_NAME_SCOPE_DEFAULT
+#define SQL_SS_NAME_SCOPE_DEFAULT SQL_SS_NAME_SCOPE_TABLE
+#endif
+#ifndef SQL_CA_SS_BASE
+#define SQL_CA_SS_BASE 1200
+#endif
+#ifndef SQL_CA_SS_TYPE_NAME
+#define SQL_CA_SS_TYPE_NAME (SQL_CA_SS_BASE + 27)
+#endif
+
```

Comment:
A comment with source (i.e. is it  Windows SDK header file, what's the name) of these constants is necessary

Reply:
Add comments. I'll squash all commits, after review complete.

Reply:
No need to squash manually.
We hit the [Squash and commit](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges) button.